pylint_import_requirements/__init__.py

```diff
@@ -32,6 +31,22 @@
 _REQUIRES_INSTALL_PREFIX = "pylint-import-requirements:"
 
 
+def _isort_place_module(module_name: str) -> str:
+    # pylint: disable=no-member
+    if hasattr(isort, "place_module"):  # isort >= v5
+        return isort.place_module(module_name)
+    if not hasattr(_isort_place_module, "sorter"):
+        _isort_place_module.sorter = isort.SortImports(file_contents="")
+    return _isort_place_module.sorter.place_module(module_name)
```

Comment:
This function is called every time we check for stdlib modules. This might happen quite a lot. So we should make sure this function is at least a bit optimized.

My suggestion would be to return the `isort.place_module` and `isort.SortImports().place_module` functions directly. The result can then be cached (as a member of the checker?)

Reply:
Good idea, thanks!

Could you re-review?

https://github.com/WanzenBug/pylint-import-requirements/pull/28/commits/80a0601ba8f1573f5691ee92e15b18870206e8eb